src/__tests__/calculatorModel.test.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  calculate,
+  updateAccumulator,
+  updateOperand,
+  updateOperation,
+  ICalculator,
+} from 'components/Calculator/CalculatorModel';
+import { Digit, OPERATION } from 'constants/calculator';
+
+const initialState: ICalculator = {
+  operation: null,
+  leftOperand: 0,
+  rightOperand: null,
+  accumulator: '0',
+};
+
+describe('Calculator Model', () => {
+  test('연산자를 저장할 수 있어야 한다.', () => {
+    let calculator = { ...initialState };
+
+    calculator = updateOperation(calculator, OPERATION.PLUS);
+
+    expect(calculator.operation).toBe(OPERATION.PLUS);
+  });
```

Comment:
```suggestion
  test.each([ { operation: OPERATION.PLUS } ])('연산자($operation)를 저장할 수 있어야 한다.', ({ operation }) => {
    let calculator = { ...initialState };

    calculator = updateOperation(calculator, operation);

    expect(calculator.operation).toBe(operation);
  });
```
이런식으로 `each`를 사용하면 테스트에 대한 인자를 여러개 만들어서 사용할 수 있습니다.
이를 이용하여 모든 연산자에 대해 테스트해보면 어떨까요?

Reply:
오 .. 정말 좋은 방법 같습니다. 이런 식으로도 테스트를 작성할 수 있네요.
each에 대해 조금 더 알아보고 적용해보겠습니다. 감사합니다. :)